webpack.config.js

```diff
@@ -6,14 +6,19 @@ const webpack = require('webpack');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const nodeExternals = require('webpack-node-externals');
 
+// ESM packages must be bundled because webpack will otherwise turn `require()` into `import()`
+// and make the import silently asynchronous. In most cases this breaks the import.
+const dependenciesToBundle = [
+  'os-locale', // Used exclusively in sync functions
+];
+
 module.exports = {
   // Set the webpack4 mode 'none' for compatibility with the behavior of the
   // webpack3 bundling step.
   mode: 'none',
   entry: {
     'addons-linter': './src/main.js',
   },
-  target: 'node',
```

Comment:
Webpack defaults to browserlist, if set. We have it already and it's set to `node 16`. This enables ESM support